packages/app/src/app/pages/Dashboard/Components/Repository/ImportRepository.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { useActions } from 'app/overmind';
+import { Stack, Text, Icon, Element } from '@codesandbox/components';
+import css from '@styled-system/css';
+
+export const ImportRepositoryCard: React.FC = () => {
+  const { openCreateSandboxModal } = useActions();
+
+  return (
+    <Element
+      as="button"
+      onClick={() => openCreateSandboxModal({ initialTab: 'import' })}
+      css={css({
+        display: 'flex',
+        width: '100%',
+        alignItems: 'center',
+        justifyContent: 'center',
+        fontSize: 4,
+        fontFamily: 'inherit',
+        fontWeight: 'normal',
+        color: '#e5e5e5',
```

Comment:
Can we change icon and font colors to `#999` and font-size to 13px?

Reply:
I tried to align the typography/colors to the repo cards on the page. This is how it looks with 13/#999
![image](https://user-images.githubusercontent.com/9945366/194115045-6019ab9f-c55b-46af-b1f4-f254d7f6f088.png)

Reply:
I can change the color, but I would sync the font size with the other cards in the page

Reply:
actually, nevermind, it is in sync with the text under the card (x branches) so it looks ok, will push both changes